### PR TITLE
Use core:move-up/down rather than custom select-prev/next commands

### PR DESCRIPTION
**Pull request by benogle**
I'm not sure why custom commands were being used to navigate the suggestion list. I think ideally, it should work with whatever you have mapped to core:move-up/down.

### CODE REVIEW

File: lib/suggestion-list.coffee [hunk 1]
```diff
@@ -76,14 +64,16 @@ class SuggestionList
   onDidConfirm: (fn) ->
     @emitter.on('did-confirm', fn)
 
-  selectNext: =>
+  selectNext: (e) =>
     @emitter.emit('did-select-next')
+    e.stopImmediatePropagation()
```
Comment (benogle) [inline lib/suggestion-list.coffee]: @nathansobo is this ok? It was the only way I could get the editor to not get the event. I feel like there is a better way...

Comment (nathansobo) [inline lib/suggestion-list.coffee]: Yes, it seems okay to me. There's really no other place to bind the event handler. Would prefer if you spelled out `event` though. Another thing I'm curious about is whether you need to add and remove these bindings programmatically... another approach would be to only perform the action and stop propagation if the autocomplete is already visible. That way you're not constantly adding and removing handlers.

Comment (benogle) [inline lib/suggestion-list.coffee]: Sounds good on renaming the var and moving the logic in the handler.

**Comment by xlaywan** [comment]
But now I cannot use alternative hotkeys to navigate the suggestion list. Like this:
```
   'alt-up': 'autocomplete-plus:select-previous'
   'alt-down': 'autocomplete-plus:select-next'
   'alt-enter': 'autocomplete-plus:confirm'
```

**Pull request merged**